### PR TITLE
AB2D-5465 Create shortcut test pipeline

### PR DESCRIPTION
**Pull request by oluwolenpbc**
This only runs when users create a branch prefixed with "test/"

**JIRA Tickets:**

[AB2D-42](https://jira.cms.gov/browse/[AB2D-5465](https://jira.cms.gov/browse/AB2D-5465)) - Description

### What Does This PR Do?

This PR is going to create a workflow with github actions that when users create a branch with the prefix test/ will run test only.

### What Should Reviewers Watch For?

That this will actually run test like we believe it will

### Usage/Deployment Instructions

Create a pull request with a branch named test/<insert branch name>

### Impacted External Components

n/a

### Database Changes
n/a
### Limitations
n/a
### Security Implications

- [ ] This PR adds new software dependencies
- [ ] This PR modifies or invalidates our security controls
- [ ] This PR stores or transmits data that was not stored or transmitted before

- [ ] This PR requires additional review of its security implications for other reasons

### Submitter Checklist

- [ ] This PR includes any required documentation changes, (`README`, changelog / release notes, website).
- [ ] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments.
- [ ] Code checked for PHI/PII exposure

### CODE REVIEW

**Review by devopsmatt — APPROVED** [review]
LGTM.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+name: Build and Test
+
+on:
+  pull_request:
+    branches:
+      - 'test/*'
+
+env:
+  ARTIFACTORY_URL: ${{ secrets.ARTIFACTORY_URL }}
```
Comment (janejr11) [inline .github/workflows/test.yml:9]: I don't see this secret in the repo. Won't this test fail as a result?

**Review by janejr11 — CHANGES_REQUESTED** [review]
I think we should get a passing test before merging.

File: .github/workflows/test.yml (hunk 1 above)
Comment (oluwolenpbc) [reply .github/workflows/test.yml:9]: We just got the artifactory creds and everything going to add that in today.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
+name: Build and Test
+
+on:
+  pull_request:
+    branches:
+      - 'test/*'
```
[the remark this answers is not in the document]
Comment (oluwolenpbc) [reply .github/workflows/test.yml:6]: My push back with on push is that this is a shortcut pipeline that is only being created to do testing. My line of thinking is that  only want to test on a PR that is prefixed with test/ 

Also not sure how we're going to test this github action without adding the workflow in first? @janejr11

File: .github/workflows/test.yml (hunk 1 above)
Comment (devopsmatt) [reply .github/workflows/test.yml:9]: @oluwolenpbc  Seems to me like you could create another branch off of this one, name it with the 'test/...' prefix and change to what @janejr11 proposed to test (on: push: branches: test/oluwolenpbc-patch-1) and be able to test from there.

**Review by devopsmatt — APPROVED** [review]
LGTM

**Pull request closed without merging**